src/SDKs/Azure.Core/data-plane/Azure.Core/Pipeline/HttpPipelineOptions.cs

```diff
@@ -4,21 +4,25 @@
 using System;
 using System.Collections.Generic;
 using System.ComponentModel;
+using Azure.Core.Pipeline.Policies;
 
 namespace Azure.Core.Pipeline
 {
     public class HttpClientOptions
     {
         private HttpPipelineTransport _transport = HttpClientTransport.Shared;
 
+        public HttpClientOptions()
```

Comment:
I'm guessing to disable telemetry, you have to set the policy to `null`?    I'm not sure this is as clear and discoverable as the property.   Should we consider a member here to disable/enable telemetry, even if by simply clearing or initializing the policy?

Reply:
Setting to null is the pattern we decided to use for disabling default policies in general. I can bring `DisableTelemetry` back but it's strange to have a special case only for one policy, but maybe it's because this is the one people are most interested in disabling.

Reply:
Fair point.  Given that information, I'd agree that this should follow the common pattern.  

As a side note, and maybe I'm just not in the loop because I'm working on track 1 areas, but do we have documentation around decisions anywhere?   Developer guides and such?   We've chosen not to include doc comments so, there would seem to be a lot of assumed context and knowledge that those stepping in to use `Azure.Base` for the first time are unlikely to be aware of.

Reply:
Unfortunately we don't have all the decisions well documented. We are mostly trying to implement guidelines from azure/azure-sdk and `Azure.Core` is in the state of constant prototype some areas of which are slowly moving to completion.

https://github.com/Azure/azure-sdk/tree/master/docs/design/dotnet is supposed to be the closest thing to developer guide for client SDKs.

Reply:
Disabling telemetry is not our primary scenario. I think we just need to make it possible, but we don't need to add APIs (and so add complexity to mainline scenarios) to make it super discoverable.